modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/ViewObjectDefinitions.tsx

```diff
@@ -324,15 +324,41 @@ export default function ViewObjectDefinitions({
 	useEffect(() => {
 		if (Liferay.FeatureFlags['LPS-148856']) {
 			const makeFetch = async () => {
-				API.getAllObjectFolders().then((response) => {
-					setObjectFoldersRequestInfo(response);
-					setSelectedObjectFolder(response.items[0]);
-					setLoading(false);
-				});
+				const allObjectFolders = await API.getAllObjectFolders();
+
+				setObjectFoldersRequestInfo(allObjectFolders);
 
 				const objectDefinitions = await API.getAllObjectDefinitions();
 
 				setObjectDefinitionActions(objectDefinitions.actions);
+
+				const currentUrl = new URL(window.location.href);
```

Comment:
Hey @rebsilva, I can see variables using uppercase URL as `baseResourceURL` and `importObjectDefinitionURL` .
The `currentUrl` do not follow this pattern, do you think we can go on with `currentURL`

Reply:
sure, i agree with this as well